components/dashboard/src/Login.tsx

```diff
@@ -48,36 +69,43 @@ export function Login() {
     }
 
     return (<div id="login-container" className="z-50 flex w-screen h-screen">
-        <div id="feature-section" className="flex-grow bg-gray-100 w-1/2 hidden lg:block">
-            <div id="feature-section-column" className="flex max-w-2xl h-full ml-auto pt-6">
-                <div className="flex flex-col space-y-12 pl-6 pr-24 m-auto">
-                    <div>
+        {showWelcome ? <div id="feature-section" className="flex-grow bg-gray-100 w-1/2 hidden lg:block">
+            <div id="feature-section-column" className="flex max-w-xl h-full mx-auto pt-6">
+                <div className="flex flex-col px-8 my-auto ml-auto">
+                    <div className="mb-12">
                         <img src={gitpod} className="h-8" />
                     </div>
-                    <div>
-                        <h1 className="xl:text-7xl text-5xl">Save Time<br /> with Prebuilds</h1>
+                    <div className="mb-10">
+                        <h1 className="text-5xl mb-3">Welcome to Gitpod</h1>
+                        <div className="text-gray-400 text-lg">
+                            Spin up fresh, automated dev environments for each task in the cloud, in seconds.
+                        </div>
                     </div>
-                    <div className="text-gray-400 text-lg">
-                        Gitpod continuously builds your git branches like a CI server. This means no more waiting for dependencies to be downloaded and builds to finish. <a className="underline underline-thickness-thin underline-offset-small hover:text-gray-600" href="https://www.gitpod.io/docs/prebuilds/" target="gitpod-docs">Learn more about Prebuilds</a>
+                    <div className="flex mb-10">
+                        <Item icon={code} iconSize="16" text="Always Ready&#x2011;To&#x2011;Code"/>
+                        <Item icon={customize} text="Personalize your Workspace"/>
+                        <Item icon={automate} text="Automate Your Development Setup"/>
                     </div>
-                    <div>
-                        <img src={terminal} className="h-64 -ml-8" />
+                    <div className="flex">
+                        <Item icon={prebuild} text="Continuously Prebuild Your Project"/>
+                        <Item icon={collaborate} text="Collaborate With Your Team"/>
+                        <Item icon={fresh} text="Fresh Workspace For Each New Task"/>
                     </div>
                 </div>
             </div>
-        </div>
-        <div id="login-section" className="flex-grow flex lg:w-1/2 w-full">
-            <div id="login-section-column" className="flex-grow max-w-2xl flex flex-col h-100 mx-auto lg:mx-0">
+        </div>: null}
+        <div id="login-section" className={"flex-grow flex w-full" + (showWelcome ? " lg:w-1/2" : "")}>
```

Comment:
**question**: What do you think of using the same layout as with the login, whit gravity towards center?

| Login | Welcome |
|-|-|
| ![image](https://user-images.githubusercontent.com/120486/112865147-28a63080-90c1-11eb-951b-e5b211755755.png) | ![image](https://user-images.githubusercontent.com/120486/112865152-293ec700-90c1-11eb-8b79-23cb056c8377.png) |

Reply:
To looks a little cross-eyed, no?

Reply:
Both approaches should be fine on small and medium screens. Center alignment (current welcome page) could look a bit off on large and extra large screens. Happy to keep both around and decide later which fits best.

Reply:
😅